back/lib/tasks/single_use/services/file_upload_custom_field_migration_service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Tasks::SingleUse::Services::FileUploadCustomFieldMigrationService
+  def initialize
+    @stats = { ideas: 0, file_fields: 0, ideas_updated: 0, errors: [] }
+  end
+
+  attr_reader :stats
+
+  def migrate(persist_changes)
+    Rails.logger.info 'Migrating file upload format for idea custom fields.'
+    Rails.logger.info "Persist: #{persist_changes}"
+
+    custom_field_keys = CustomField.where(input_type: 'file_upload').pluck(:key)
```

Comment:
`input_type` can also be `files`. I guess we don't need to cover it too, do we? https://github.com/CitizenLabDotCo/citizenlab/blob/4b378e48a272bd1dd14379cb6c80a9d68d3e9d7a/back/app/models/custom_field.rb#L52

Reply:
No files is only used for built in fields I think